src/components/BookIndividual.jsx

```diff
@@ -1,22 +1,36 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/booksSlice';
 
-const BookIndividual = ({ book, onDelete }) => (
-  <div className="info">
-    <h2>{book.title}</h2>
-    <p>
-      Author:
-      {book.author}
-    </p>
-    <button type="button" className="btn" onClick={() => onDelete(book.id)}>Delete</button>
-  </div>
-);
+const BookIndividual = ({ book, onDelete }) => {
+  const dispatch = useDispatch();
+
+  const handleDelete = () => {
+    dispatch(removeBook(book.item_id));
+    onDelete(book.item_id);
+  };
```

Comment:
- [ ] Delete button is working fine for default books available in store but if I add books from the book and try to delete any books It's deleting two books that are added from the forms. Please make sure to fix this, (To reproduce this, Try to add more than one book from the form and try to delete any of the books).